boefjes/boefjes/docker_boefjes_runner.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from datetime import datetime, timezone
+
+import docker
+from docker.errors import APIError, ContainerError, ImageNotFound
+from requests import HTTPError
+
+from boefjes.clients.bytes_client import BytesAPIClient
+from boefjes.clients.scheduler_client import SchedulerAPIClient, TaskStatus
+from boefjes.config import settings
+from boefjes.job_models import BoefjeMeta
+from boefjes.katalogus.models import Boefje
+
+logger = logging.getLogger(__name__)
+
+
+class DockerBoefjesRunner:
```

Comment:
I think this class means to extend the `BoefjeJobRunner` interface, then later to clean things up we could introduce a more advanced runner that switches between the right implementation per task